packages/office-addin-dev-settings/.vscode/launch.json

```diff
@@ -142,7 +142,7 @@
       "type": "node",
       "request": "launch",
       "program": "${workspaceFolder}/lib/cli.js",
-      "args": ["sideload", "c:\\dev\\git\\Office-Addin-TaskPane\\manifest.excel.xml", "web", "-d", "https://1drv.ms/x/s!Au78PotIZz2Q5OEmJ-xvfBs3ODNUYg?e=F64J5i"]
```

Comment:
What is this change?

Reply:
This changes the debug launching for office-addin-dev-settings to a manifest that is part of the project rather than assuming another location on the dev box outside of the current project.